js_modules/dagster-ui/packages/ui-core/src/runs/usePaginatedRunsTableRuns.tsx

```diff
@@ -7,7 +7,7 @@ import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
 import {PAGE_SIZE} from '../assets/AutoMaterializePolicyPage/useEvaluationsQueryResult';
 import {RunsFilter} from '../graphql/types';
 
-export function usePaginatedRunsTableRuns(filter: RunsFilter) {
+export function usePaginatedRunsTableRuns(filter: RunsFilter, pageSize: number = PAGE_SIZE) {
```

Comment:
Ahh, the `PAGE_SIZE` value being imported here is definitely not correct -- `useEvaluationsQueryResult` is totally unrelated to this code, so we shouldn't be using its `PAGE_SIZE` at all. My guess is that this is a rogue auto-import from when this file was created.

Let's go ahead and require the caller to provide a value. That should ensure that the caller's specified page size always matches the query arguments.

```suggestion
export function usePaginatedRunsTableRuns(filter: RunsFilter, pageSize: number) {
```

You may need to update one or two more callsites in order to get TypeScript and tests to pass.